test/controllers/EmailControllerSpec.scala

```diff
@@ -59,19 +72,33 @@ class EmailControllerSpec extends SpecBase {
         status(result) mustBe OK
       }
     }
+
+    "display undeliverableEmail page when exception occurs while connector is making the API call" in new Setup {
+      when(mockConnector.verifiedEmail(any)).thenReturn(Future.successful(emailVerifiedResponseWithNoEmailId))
```

Comment:
do we want a new line between the test name and the first when? like in the test above? if so add one otherwise remove the line from the above test (line 50)

Reply:
Added a line break